extensions/extensions-docker/aissemble-model-training-api-containers/aissemble-model-training-api-sagemaker/src/main/resources/docker/Dockerfile

```diff
@@ -37,7 +31,8 @@ LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
 
 # Set/move any required environmental variables and key scripts/configs needed by FastAPI from the base image
 # into the slimmed down final Python image
-ENV API_MAIN=model_training_api_sagemaker.model_training_api_sagemaker
-COPY --from=builder --chown=1001 /start.sh /start.sh
+ENV API_HOST='0.0.0.0'
+ENV API_PORT='80'
+ENV EXTRA_OPTS=''
 
-CMD ["/start.sh"]
+CMD python -m uvicorn "model_training_api_sagemaker.model_training_api_sagemaker:app" --host "$API_HOST" --port "$API_PORT" $EXTRA_OPTS
```

Comment:
Q: Is this just the contents of /start.sh from the old base image?

Reply:
Functionally, yes.  There was an envvar that could be set to enable proxy headers, that I replaced with a more generic "EXTRA_OPTS".